File: src/main/java/dqualizer/research/dqapi/DqApiApplication.java
package dqualizer.research.dqapi;

import dqualizer.research.dqapi.repositories.RqaDefinitionRepository;
import dqualizer.research.dqapi.models.rqa.RqaDefinition;
import dqualizer.research.dqapi.models.rqa.RuntimeQualityAnalysis;
import dqualizer.research.dqapi.models.rqa.enums.Environment;
import dqualizer.research.dqapi.models.rqa.loadtest.Loadtest;
import dqualizer.research.dqapi.services.RuntimeQualityAnalysisService;
import org.springframework.boot.CommandLineRunner;
import org.springframework.boot.SpringApplication;
import org.springframework.boot.autoconfigure.SpringBootApplication;
import org.springframework.context.annotation.Bean;

import java.util.ArrayList;
import java.util.List;

@SpringBootApplication
public class DqApiApplication {

	public static void main(String[] args) {
		SpringApplication.run(DqApiApplication.class, args);
	}

}

File: src/main/kotlin/io/github/dqualizer/dqapi/DqapiApplication.kt
package io.github.dqualizer.dqapi

import org.springframework.boot.autoconfigure.SpringBootApplication
import org.springframework.boot.runApplication
import org.springframework.data.mongodb.repository.config.EnableMongoRepositories

@EnableMongoRepositories
@SpringBootApplication
class DqapiApplication

fun main(args: Array<String>) {
  runApplication<DqapiApplication>(*args)
}
